e2e/app-with-domain/src/index.js

```diff
@@ -0,0 +1,31 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+let express = require('express');
+
+let app1 = express();
```

Comment:
does `app.listen()` block in js? if not - can we group the app1/app2 stuff logically? like:
```
const app1 = express();
app1.get("/", etc...)
app1.listen(port, etc...)

const app2 = express();
// etc
```